File: csharp/SlimDownYourAggregates.Tests/Slimmed/Entities/CommitteeApproval.cs
namespace SlimDownYourAggregates.Tests.Slimmed.Entities;

public class CommitteeApproval
{
    public CommitteeApproval(bool isApproved, string feedback)
    {
        IsApproved = isApproved;
        Feedback = feedback;
    }

    public bool IsApproved { get; }
    public string Feedback { get; }
}

File: java/src/main/java/io/eventdriven/slimdownaggregates/shorter/original/entities/CommitteeApproval.java
package io.eventdriven.slimdownaggregates.shorter.original.entities;

public class CommitteeApproval {
  private final boolean isApproved;
  private final String feedback;

  public CommitteeApproval(boolean isApproved, String feedback) {
    this.isApproved = isApproved;
    this.feedback = feedback;
  }

  public boolean isApproved() {
    return isApproved;
  }

  public String getFeedback() {
    return feedback;
  }
}
